src/common/match.c

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-2.0-only
+#define _POSIX_C_SOURCE 200809L
+#include <glib.h>
```

Comment:
The glib include could be removed here as its now included in the header

Reply:
Done.